packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -382,44 +357,39 @@ class Treemap extends StatelessWidget {
   /// ----------------------------
   /// ```
   Widget buildTreemap(BuildContext context) {
-    if (levelsVisible > 0 && rootNode.children.isNotEmpty) {
+    if (rootNode.children.isNotEmpty) {
       return Padding(
         padding: const EdgeInsets.all(1.0),
-        child: Column(
-          children: [
-            if (height > minHeightToDisplayTitleText) buildTitleText(context),
-            Expanded(
-              child: Treemap.fromNodes(
-                nodes: rootNode.children,
-                levelsVisible: levelsVisible,
-                isOutermostLevel: isOutermostLevel,
-                onRootChangedCallback: onRootChangedCallback,
-                height: height,
+        child: buildSelectable(
+          selectable: !isOutermostLevel,
+          child: Column(
+            children: [
+              if (height > minHeightToDisplayTitleText) buildTitleText(context),
+              Expanded(
+                child: Treemap.fromNodes(
+                  nodes: rootNode.children,
+                  levelsVisible: levelsVisible,
+                  isOutermostLevel: isOutermostLevel,
+                  onRootChangedCallback: onRootChangedCallback,
+                  width: width,
+                  height: height,
+                ),
               ),
-            ),
-          ],
+            ],
+          ),
         ),
       );
     } else {
       return Column(
         children: [
           Expanded(
-            child: buildSelectable(
-              child: Container(
-                decoration: BoxDecoration(
-                  color: rootNode.displayColor,
-                  border: Border.all(color: Colors.black87),
-                ),
-                child: Center(
-                  child: height > minHeightToDisplayCellText
-                      ? buildNameAndSizeText(
-                          textColor:
-                              rootNode.showDiff ? Colors.white : Colors.black,
-                          oneLine: false,
-                        )
-                      : const SizedBox(),
-                ),
+            child: CustomPaint(
```

Comment:
Can't this stay the same and just return a widget via `buildSelectable`? this isn't the level we are painting for other nodes. Other nodes at this level are widgets whose children are painted.

Reply:
Done